tests/deploy/k8s-cloudbeat-tests/templates/cloudbeat-ds.yml

```diff
@@ -5,15 +5,18 @@ metadata:
   name: cloudbeat
   namespace: kube-system
   labels:
+    catf: related
```

Comment:
What's this for?

Reply:
These labels are used to identify deployments related to catf (cloudbeat automated tests framework)